### ISSUE TITLE
In flutter web some emoji is not visible 

### ISSUE THREAD

**Issue by sinnoorc** [Fintasys/emoji_picker_flutter#155]
<img width="354" alt="Screenshot 2023-09-25 at 12 28 35 PM" src="https://github.com/Fintasys/emoji_picker_flutter/assets/62532648/b7abc72a-f4fa-483b-953b-a4284e354fa7">

**Comment by Fintasys** [comment]
Can you provide more information? Did you read the section in the readme about custom emoji and checked out the example?
https://github.com/Fintasys/emoji_picker_flutter/blob/master/example/lib/main_custom_font.dart

**Comment by jehhxuxu** [comment]
I'm having the same problem here with Flutter Web, and I don’t have custom emojis, the default ones disappeared...

<img width="337" alt="Screenshot 2023-10-14 at 14 34 23" src="https://github.com/Fintasys/emoji_picker_flutter/assets/27310692/4c450eb1-2b28-457d-8dbe-8a413fc37d3d">

**Comment by danielGz** [comment]
It's not a thing of the emojis not existing in the OS, they do appear in the textField after selecting them, I think flutter web has issues on how it renders paths in the same widget as the emoji:
![image](https://github.com/Fintasys/emoji_picker_flutter/assets/26932503/accb1dfb-c50a-46d9-842c-7f8326f5dfce)

vs
![image](https://github.com/Fintasys/emoji_picker_flutter/assets/26932503/ba0f2f4b-29e5-4c4e-b10d-b15dbfca1e1b)

I have modified the package obviously, as you can see removed the triangle at the corner of each cell, I also simplified the cell for the skin tone overlay, removed the inkwell and elevation.

**Comment by Fintasys** [comment]
I tried both renderer for web with the `example/main.dart` (without custom font) and didn't have any issues. Can you point out which part removing brought improvements or how to replicate? 
<img width="680" alt="Screen Shot 2023-10-16 at 22 27 47" src="https://github.com/Fintasys/emoji_picker_flutter/assets/11667234/db0445ec-8f97-4b55-973c-9bef9ae76d10">
<img width="677" alt="Screen Shot 2023-10-16 at 22 28 26" src="https://github.com/Fintasys/emoji_picker_flutter/assets/11667234/63801761-ee27-4fca-9bb8-08d9cd8320b7">

**Comment by jehhxuxu** [comment]
About the browser, I was testing with Chrome... and I can’t get the skin tone as well =/

**Comment by Fintasys** [comment]
Any idea how to reproduce the issue? Difficult to check the issue otherwise 😢

**Comment by Fintasys** [comment]
Please re-open the issue if it's still happening 🙏

**Comment by GleammerRay** [comment]
Same issue on my end

**Comment by GleammerRay** [comment]
Apparently there's a bunch of issues with Flutter's HTML renderer (which is why https://github.com/flutter/flutter/issues/145954 exists) so I moved my extension over to CanvasKit and it seems to have fixed the emojis.

There were some CSP issues on my end so I ended up employing the option implemented in https://github.com/flutter/flutter/issues/70101 to bundle CanvasKit into my extension.

I built and ran successfully with `flutter build web --web-renderer canvaskit --csp --pwa-strategy=none --source-maps --dart-define=FLUTTER_WEB_CANVASKIT_URL=/`.

I also had to change my CSP from:
```json
"extension_pages": "script-src 'self' ; object-src 'self'"
```
To:
```json
"extension_pages": "script-src 'self' 'wasm-unsafe-eval' ; object-src 'self'"
```

Apparently CanvasKit renders emojis in black and white by default, so I had to change a line in my web index files from this:
```js
let appRunner = await engineInitializer.initializeEngine();
```
To this:
```js
let appRunner = await engineInitializer.initializeEngine({useColorEmoji: true});
```
(see https://docs.flutter.dev/platform-integration/web/initialization)

I have yet to test the extension on Firefox but it seems to work on Chromium so far.

Flutter web is still quite a mess. 😅
